src/main/kotlin/no/nav/familie/ef/sak/api/beregning/BeregningController.kt

```diff
@@ -29,18 +29,18 @@ class BeregningController(private val stegService: StegService,
     fun beregnYtelseForStønad(@PathVariable behandlingId: UUID, @RequestBody beregningRequest: BeregningRequest): Ressurs<UUID> {
         tilgangService.validerTilgangTilBehandling(behandlingId)
         val behandling = behandlingService.hentBehandling(behandlingId)
-        val fagsak = fagsakService.hentFagsak(behandling.fagsakId)
+        val aktivIdent = fagsakService.hentAktivIdent(behandling.fagsakId)
```

Comment:
Kan vi droppe fagsakService og bruke behandlingService direkte?
Isåfall trenger vi ikke å hente behandlingen heller?

Reply:
Jo! 👍